website/docusaurus.config.js

```diff
@@ -0,0 +1,185 @@
+// @ts-check
+// Note: type annotations allow type checking and IDEs autocompletion
+
+const lightCodeTheme = require("prism-react-renderer/themes/github");
+const darkCodeTheme = require("prism-react-renderer/themes/dracula");
+
+const TITLE = "Wallet-API";
+
+/** @type {import('@docusaurus/types').Config} */
+const config = {
+  title: TITLE,
+  tagline:
+    "Integrate your application or service inside a Ledger client wallet",
+  url: "https://your-docusaurus-test-site.com",
```

Comment:
We can maybe update the url now ?

Reply:
But done anyway -> b40858e163a17e722ff5a9fe2549c1a71efd91b4 👍